src/amo/components/AddonMeta.js

```diff
@@ -1,37 +0,0 @@
-import React, { PropTypes } from 'react';
```

Comment:
I don't think we should delete this file yet. I know it has been pushed to V2 but Jorge requested that we at least implement user count for V1 since we do have that data available to us: https://docs.google.com/document/d/13g8tmoII8oCNrUXOl2HegsQst7WEsXukBnIqPUNrqb0/edit#heading=h.2ltzvz63pc6h

Also, I was planning to get the user count stuff done today :) https://github.com/mozilla/addons-frontend/issues/1293